app/ts/components/subcomponents/ConfigureRpcConnection.tsx

```diff
@@ -21,9 +25,55 @@ const ConfigureRpcContext = createContext<ConfigureRpcContext | undefined>(undef
 const RpcQueryProvider = ({ children }: { children: ComponentChildren }) => {
 	const { value: rpcQuery, waitFor, reset: resetRpcQuery } = useAsyncState<Network>()
 
+	const checkServerAvailability = async (url: string) => {
+		try {
+			const provider = new JsonRpcProvider(url)
+			return await provider.getNetwork()
+		} catch {
+			throw new Error('Server should be reachable')
+		}
+	}
+
+	const validateEthSimulateSupport = async (url: string) => {
+		// create temporary rpc server from an RPC URL
+		const interimRpc: RpcEntry = { httpsRpc: url, name: '', chainId: 0n, currencyName: '', currencyTicker: '', primary: false, minimized: false, }
+
+		// test eth_simulate request
+		const requestHandler = new EthereumJSONRpcRequestHandler(interimRpc, true)
+		const ethClient = new EthereumClientService(requestHandler, async () => { }, async () => { })
+		ethClient.getChainId
+		const testBlockCall = {
+			stateOverrides: {
+				'0xc000000000000000000000000000000000000000': {
+					balance: 5120000000n
+				}
+			},
+			calls: [
+				{
+					from: 0xc000000000000000000000000000000000000000n,
+					to: 0x1000000000000000000000000000000000000000n,
+					value: 1n,
+					maxFeePerGas: 15n
+				}
+			]
+		} satisfies BlockCalls
+
+		try {
+			const simulationResult = await ethClient.ethSimulateV1([testBlockCall], 'latest', undefined)
+			const serializedResult = serialize(ethSimulateV1Result, simulationResult)
+			ethSimulateV1Result.parse(serializedResult)
+		} catch (error) {
+			let errorMessage = 'Error encountered while validating eth_simulateV1 support'
+			if (error instanceof Error) errorMessage = `${errorMessage} (${error.message})`
+			console.warn(errorMessage)
```

Comment:
I feel you should warn with the error instead of with the errormessage crafted by us, that helps to debug if there's something wrong

Reply:
changed message to `RPC eth_simulateV1 validation error` and included the error object on the warning for support debugging